src/Generator/Generator.php

```diff
@@ -234,14 +238,27 @@ public function generate(MapperGeneratorMetadataInterface $mapperGeneratorMetada
                 continue;
             }
 
-            $transformer = $propertyMapping->transformer;
-
             $fieldValueVariable = new Expr\Variable($uniqueVariableScope->getUniqueName('fieldValue'));
             /** Create expression on how to read the value from the source */
             $sourcePropertyAccessor = new Expr\Assign($fieldValueVariable, $propertyMapping->readAccessor->getExpression($sourceInput));
 
-            /* Create expression to transform the readed value into the wanted writed value, depending on the transform it may add new statements to get the correct value */
-            [$output, $propStatements] = $transformer->transform($fieldValueVariable, $result, $propertyMapping, $uniqueVariableScope);
+            $transformer = $propertyMapping->transformer;
+
+            if (is_string($transformer)) {
+//                $output = new Expr\MethodCall(
+//                    new Expr\Variable('customTransformerRegistry'),
+//                    'transform',
+//                    [
+//                        new Arg(new Scalar\String_($customTransformer)),
+//                        new Arg($fieldValueVariable),
+//                    ]
+//                );
+                $propStatements = [];
+                $output = $this->customTransformerExporter->extract($transformer, $fieldValueVariable);
```

Comment:
A quick comment to explain this is when you have a CustomTransformer and needs to parse the transformer extract method to know what to add within the Transformer